mkosi/__init__.py

```diff
@@ -583,7 +583,7 @@ def copy_fd(oldfd: int, newfd: int) -> None:
     try:
         _reflink(oldfd, newfd)
     except OSError as e:
-        if e.errno not in {errno.EXDEV, errno.EOPNOTSUPP}:
+        if e.errno not in {errno.EXDEV, errno.EOPNOTSUPP, errno.ENOTTY}:
```

Comment:
As a general note (doesn't need to be changed): While in general sets are preferable for these checks, due to O(1) access, for such small sets tuples beat them due to memory locality (at least they did, when I last benchmarked this)